app/konnect/org-management/okta-idp.md

```diff
@@ -191,7 +196,33 @@ application into {{site.konnect_saas}}.
     * The path does not require a slash (`/`).
 
 1. Click **Save**.
+{% endnavtab %}
+{% navtab SAML %}
+1. In another separate browser tab, log in to [{{site.konnect_saas}}](https://cloud.konghq.com).
+2. Click {% konnect_icon organizations %} **Organization**, and then **Auth Settings**.
+3. Click **Configure provider** for **SAML**.
+
+4. In Okta, locate your Metadata URL:
+    1. Go to **Sign On** page in the Okta application created in the previous step.
+    2. Copy the **Metadata URL** under the Settings section. It should look something like:
+
+        `
+        https://<your-okta-domain>.okta.com/app/exkgzjkl0kUZB06Ky5d7/sso/saml/metadata
+        `
+5. In the **Organization Login Path** box, enter a unique string. For example: `examplepath`.
+
+   {{site.konnect_short_name}} uses this string to generate a custom login
+   URL for your organization.
+
+   Requirements:
+    * The path must be unique *across all {{site.konnect_short_name}} organizations*.
+      If your desired path is already taken, you must choose another one.
+    * The path can be any alphanumeric string.
+    * The path does not require a slash (`/`).
 
+6. Click **Save**.
```

Comment:
After clicking save, there the user will have to go back to Okta to finish configuring the SAML application and [fill in those placeholders](https://github.com/Kong/docs.konghq.com/pull/7481/files#diff-c63b47a1b4b067ce69a603b656b6b5b54b4775cf9a9f68b956d49340131c304cR57). Our UI should provide the values to use after clicking Save in Konnect.